source/install/requirements.rst

```diff
@@ -26,12 +26,21 @@ Client Software
 PC Web Experience
 ^^^^^^^^^^^^^^^^^
 
--  PC: Windows 7, Windows 8, Windows 10 with IE 11*, Chrome 43+, Firefox 52+, and Edge 40+ (or EdgeHTML v15+)
--  Mac: OS 10 (Safari 9, Chrome 43+)
+-  PC: Windows 7+ with IE 11*, Chrome 43+, Firefox 52+, and Edge 40+ (or EdgeHTML v15+)
```

Comment:
We should update this to versions we actually know we support. So Chrome 59+ (which is the version electron is on). As well as Firefox 60+ (Current Firefox ESR).

Reply:
Not sure what we mean by edge 40+ because that's not a version of edge.  Looking at the caniuse. I think the earist version of Edge we can support is 16. So 16+ for edge. Same for EdgeHTML we support 16+.

Reply:
40+ was based off of https://en.wikipedia.org/wiki/Microsoft_Edge#Release_history. 

Re: EdgeHTML/Chrome/Firefox versions: While what you say makes sense (we shouldn't say we support Chrome v43 if we have no idea if Mattermost still works with that version), I think there should be a broader process via QA to validate we still support version XX, and if not, make the call on dropping support for it or fixing the issue to support version XX

Reply:
And if we drop support for older versions, we should release note it somewhere.

Reply:
(Will post in QA to get their thoughts)

Reply:
Ah sorry I read stuff wrong. All the stuff I was looking refers to edge by the EdgeHTML version. So we support Edge 41+ (which is EdgeHTML 16+).

Reply:
@crspeller The more time I spend looking into this, the more I feel it’s not critical enough to go through the process of updating these version numbers right now.

1 - We have a UI page that asks people to updated their browsers if they’re on an old version we no longer support.

That means a change to that page, and blocking some % of users accessing Mattermost.

![image](https://user-images.githubusercontent.com/13119842/45397805-ab546680-b60f-11e8-9488-1de9e50032f4.png)

2 - We’d need to release note it so customers know they must upgrade their users’ browsers if they’re not on one of the supported versions (or else their users will be blocked from Mattermost).

This caused a lot of trouble for at least one customer last time we did it for Firefox and decided to drop v45 ESR support, because we legitimately could no longer support it.

A lot of our customer base is in government organizations and financial institutions which are notorious for running old browser versions.

3 - It’s not clear what benefit we get out of this change. No one is hitting issues with it at the moment.

4 - It’s not clear what impact the changes would have without further investigation (e.g. I looked into Firefox usage and v52 ESR is used more than v60 ESR, at least by Mattermost users)

5 - Competitors have lenient browser version support, see Slack’s attached 

![image](https://user-images.githubusercontent.com/13119842/45397983-6977f000-b610-11e8-888c-c84b54a5805a.png)

----

There is a separate conversation we've had with QA about this, and we're looking at options to more easily automate UI testing across browsers and versions, whether with NightWatchJS or something else.

But unless you feel really strongly about this, I don't think it's worth updating this right now.

Reply:
1. We don't have to hard block, we can just update the numbers here. 
2. We can release note it sure. 
3. Actually having an accurate description of what we support? There isn't much point in listing support for browser versions we don't actually support. Also having a definite and updated browser support versions will allow us to improve performance by using modern browser features. 
4. It's not a hard block. So people running old versions should still work, but FF doesn't support their old ESR anymore so there is little reason for us to.
5. They have more resources. Also their support is not much more lenient then ours.

Reply:
@crspeller I didn't realize Firefox v52ESR was no longer supported as of Sept 5th. That makes it easier to push people to v60+.

Alright, so we would change version support for Chrome (v59+), Firefox (v60+) and Edge (v41+).
 - Have touched base with a customer who might be affected and see if they'd be open to upgrading their browser versions given FFv52 is considered EOL.
 - I'll also see if we can drop IE11 mobile view support at the same time, since it's buggy and likely not used. Have touched with a customer who might be affected on that.

PS: If it's not a hard block, then we'd have people running on old unsupported browsers with potentially a poor user experience. That said, we indirectly do that already so that's less of a concern.

Reply:
Customers I talked to are okay with these changes, next I've shared a proposal with QA: https://pre-release.mattermost.com/core/pl/sb4fq6qhyfbb5xjdp7x3ud146e